### DISCUSSION TITLE
[request] optional option to keep emoji window open

### DISCUSSION THREAD

**Discussion by lastmetroid** [night/betterttv#6931] [Ideas]
Please give optional option to keep the emoji window open until you send the message, instead of holding shift. Often you want to spam multiple emojis in a row , it is annoying the emoji window closes when you click one emoji with the mouse only and have to hold shift.

suggestion:

- give optional option to keep the window open until you send the message
- close when you click out / focus changes

**Comment by night** [comment]
We have no plans to add an option for this. We used to offer this as an explicit setting, but instead it makes more sense to require you to hold Shift given how infrequently the option was used.
